_static/scripts/part2/simplecache/simple_cache.cc

```diff
@@ -386,7 +374,7 @@ SimpleCache::insert(PacketPtr pkt)
 
         // Write back the data.
         // Create a new request-packet pair
-        RequestPtr req = new Request(block->first, blockSize, 0, 0);
+        RequestPtr req(new Request(block->first, blockSize, 0, 0));
```

Comment:
Why this change?

Reply:
The original simple_cache.cc will generate the following error when I compile, so I change it.
```
[     CXX] X86/learning_gem5/simple_cache/simple_cache.cc -> .o
build/X86/learning_gem5/simple_cache/simple_cache.cc:378:14: error: no viable conversion from 'Request *' to 'RequestPtr' (aka 'shared_ptr<Request>')
RequestPtr req = new Request(block->first, blockSize, 0, 0);
```

Reply:
Ok.